### PR TITLE
Fix downloading for masterani

### PR DESCRIPTION
**Pull request by freddiaN**
This should fix downloading from masterani/mp4upload for some shows (since it also seems that some anime, like Koe No Katachi, are not affected by broken mp4upload SSL certs).

### CODE REVIEW

File: anime_downloader/sites/anime.py [hunk 1]
```diff
@@ -102,6 +102,7 @@ class BaseEpisode:
     QUALITIES = None
     title = ''
     stream_url = ''
+    ssl = True
```
Comment (vn-ki) [inline anime_downloader/sites/anime.py]: You don't seem to be using these. Remove them.

File: anime_downloader/sites/masterani.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
 class MasteraniEpisode(BaseEpisode):
     QUALITIES = ['360p', '480p', '720p', '1080p']
+    ssl = False
```
Comment (vn-ki) [inline anime_downloader/sites/masterani.py]: Same as above. Remove since you are not using them.

File: anime_downloader/util.py [hunk 3]
```diff
@@ -75,7 +75,6 @@ def search(query, provider):
 
     return url
 
-
```
Comment (vn-ki) [inline anime_downloader/util.py]: Don't remove spaces. :)

File: anime_downloader/sites/anime.py [hunk 4]
```diff
@@ -166,7 +167,7 @@ def _get_sources(self):
         raise NotImplementedError
 
     def download(self, force=False, path=None,
-                 format='{anime_title}_{ep_no}', range_size=None):
+                 format='{anime_title}_{ep_no}', range_size=None, ssl=ssl):
```
Comment (vn-ki) [inline anime_downloader/sites/anime.py]: `ssl=True`, right?

File: anime_downloader/cli.py [hunk 5]
```diff
@@ -140,7 +140,14 @@ def dl(ctx, anime_url, episode_range, url, player, skip_download, quality,
             if chunk_size is not None:
                 chunk_size *= 1e6
                 chunk_size = int(chunk_size)
-            episode.download(force=force_download,
+            if cls == Masterani:
```
Comment (vn-ki) [inline anime_downloader/cli.py]: Do 
```
cls.__name__ == 'Masterani'
```

File: anime_downloader/cli.py [hunk 6]
```diff
@@ -4,6 +4,7 @@
 
 import logging
 
+from anime_downloader.sites.masterani import Masterani
```
Comment (vn-ki) [inline anime_downloader/cli.py]: Remove this import as you won't need it anymore after the change mentioned below.

**Comment by freddiaN** [comment]
updated my fork to fix those issues (and also tested again, still worked)

**Pull request merged**